backend/README.md

```diff
@@ -29,19 +29,24 @@ registers and logins user
 returns:
 `{ "key": "some-string" }`
 
-### /pages/
+### /api/users/pages/
```

Comment:
So I imagine we need to update the urls in the actions to reflect this?

Reply:
Yes, we will need to do so.